packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -429,20 +429,26 @@ class DevFS {
           assetPathsToEvict.add(archivePath);
       }
     });
+    if (generator != null) {
+      // We run generator even if [dirtyEntries] was empty because we want
+      // to keep logic of accepting/rejecting generator's output simple:
+      // we must accept/reject generator's output after every [update] call.
+      // Incremental run with no changes is supposed to be fast (considering
+      // that it is initiated by user key press).
+      final List<String> invalidatedFiles = <String>[];
+      for (DevFSContent content in dirtyEntries.values)
+        if (content is DevFSFileContent)
+          invalidatedFiles.add(content.file.uri.toString());
+      printTrace('Compiling dart to kernel with ${invalidatedFiles.length} updated files');
```

Comment:
printTrace for debugging? can it be removed?

Reply:
Not sure what are the guidelines for printTraces, but I think this one is helpful - between it and next `printTrace('Update files')` we can see for example how much time is spent in actual compilation.